src/components/rmrk/Gallery/Pagination.vue

```diff
@@ -50,6 +63,13 @@ export default class Pagination extends Vue {
     }
   }
 
+  public goToRandomPage() {
+    this.onPageChange()
+    const pageSize = Math.floor(this.total / this.perPage)
+    let randomNumber = getRandomIntInRange(1, pageSize)
```

Comment:
Here I would have simply written
```js
let randomNumber = Math.floor(Math.random() * pageSize) + 1
```
but sure we can keep this function

Reply:
Have kept this function